source/crud/tests/v1/write/updateMany-collation.json

```diff
@@ -14,6 +14,7 @@
     }
   ],
   "minServerVersion": "3.4",
+  "serverless": "forbid",
```

Comment:
Same behavior with the operation being an individual request of `bulkWrite` operation.